packages/integration-sdk-cli/src/commands/run.ts

```diff
@@ -87,6 +101,12 @@ export function run() {
         });
 
         log.displaySynchronizationResults(abortResult);
+      } finally {
+        logger.publishMetric({
+          name: 'total-duration',
+          value: Date.now() - startTime,
+          unit: 'Milliseconds',
```

Comment:
I'm interested in learning more about the metric names you've already thought about, or whether there is a metric naming standard you think we should adopt (there are [many to choose from](https://www.google.com/search?q=metrics+naming+convention&oq=naming+metrics&aqs=chrome.2.69i57j0l4.5870j0j7&sourceid=chrome&ie=UTF-8)). I think this is worth discussion early on. For example, I would prefer `duration-total`, so that all `duration-*` metrics are grouped instead of all `total-*` things across different metrics (like `total-requests` or `total-steps`).

Reply:
Okay, I'm down with changing the metrics to be prefixed with the type of thing they are tracking.